include/deal.II/base/numbers.h

```diff
@@ -81,13 +81,13 @@ namespace internal
      * Maximal vector length of VectorizedArray for double.
      */
     constexpr static unsigned int max_width =
-#if DEAL_II_COMPILER_VECTORIZATION_LEVEL >= 1 && defined(__ALTIVEC__)
+#if DEAL_II_VECTORIZATION_WIDTH_IN_BITS >= 128 && defined(__ALTIVEC__)
       2;
-#elif DEAL_II_COMPILER_VECTORIZATION_LEVEL >= 3 && defined(__AVX512F__)
+#elif DEAL_II_VECTORIZATION_WIDTH_IN_BITS >= 512 && defined(__AVX512F__)
```

Comment:
are these ``&& defined(bla)`` necessary? I thought we catch this in vectorization.h

Reply:
You are right, we should be getting into these errors here: https://github.com/dealii/dealii/blob/61a022c0784163310939baa4dbde83971b89faa7/include/deal.II/base/vectorization.h#L46-L55
So I will simply remove the second part.